index.bs

```diff
@@ -2417,6 +2432,26 @@ To remove the need to parse CBOR at all in many cases, {{AuthenticatorAttestatio
 
 Note: {{AuthenticatorAttestationResponse/getPublicKey()}} and {{AuthenticatorAttestationResponse/getAuthenticatorData()}} were only added in level two of this spec. [=[RPS]=] SHOULD use feature detection before using these functions by testing the value of `'getPublicKey' in AuthenticatorAttestationResponse.prototype`. [=[RPS]=] that require this function to exist may not interoperate with older user-agents.
 
+
+#### Determining Authenticator Attachment from Credential Transports #### {#sctn-attachments-from-transports}
```

Comment:
```suggestion
### Determining Authenticator Attachment from Authenticator Transports ### {#sctn-attachments-from-transports}
```
...and additionally move this section up into section 5.1, perhaps after  `sctn-isUserVerifyingPlatformAuthenticatorAvailable`.